doc/lua_api.txt

```diff
@@ -1481,7 +1481,8 @@ Usage
 -----
 
 Groups are stored in a table, having the group names with keys and the
-group ratings as values. For example:
+group ratings as values. Group ratings are integer values within the
```

Comment:
Also mention:

- A group rating of `0` means that the group is not set.
- Dig groups must be 1 or more